src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -340,6 +316,7 @@ def set_consent_preferences(
         else:
             preference_dict = dict(preference)
             preference_dict["provided_identity_id"] = provided_identity.id
+            preference_dict.pop("executable")
```

Comment:
I felt like I had to do some weird things because `executable` was in `ConsentSchema` but not the ORM `Consent` object

Reply:
Yes, good point, this is not feeling ideal given the number of locations you've had to change.  

So a new approach: What about if we instead did something like:

- Left the `ConsentSchema` untouched
- Modified the `ConsentPreferencesWithVerificationCode` schema to have a new `executable_options` key
- `executable_options` might just be a list of new `ExecutableConsent` (new schema) with a data_use and executable (bool) fields.
- Pass in the executable options to`queue_privacy_request_to_propagate_consent` 
- Use those instead of what I was pulling from the config.json to filter which Consent options are saved on the privacy request
- I'm thinking we still just save the executable preferences to the PrivacyRequest for now.
- Still don't queue the privacy request if we don't have executable preferences

Then hopefully the processing of saving Consent records to the db is untouched and _prepare_consent_preferences is untouched and the PrivacyRequest still has just executable preferences.